README.md

```diff
@@ -344,8 +344,35 @@ For a comparison of some of these, see this
 
 ## Storage Backends
 
-In order to not download or scan any previously scanned sources again, the _scanner_ can use a storage backend to store
-scan results for later reuse.
+In order to not download or scan any previously scanned sources again or to reuse scan results generated via other
```

Comment:
Maybe a comma after "again"? I'm usure about the official English grammar rules, but it reads better to me to separate the mentioned use-case from the first "or" in the sentence.